### ISSUE TITLE
Error: LogBox must be enabled before AppContainer is required so that it can properly wrap the console methods.

### ISSUE THREAD

**Issue by linbo8303** [magicismight/react-native-root-toast#126]
**react-native-root-toast** conflict with **LogBox**

![Simulator Screen Shot - iPhone 11 - 2020-05-07 at 09 44 26](https://user-images.githubusercontent.com/15687159/81239732-ff178d00-9048-11ea-9685-fc4f3d392e96.png)

**Comment by Tejpbit** [comment]
I'm assuming you've put the `require('react-native').unstable_enableLogBox();` at the top of the index.js file? Didn't work for me either.

What worked for me was creating a "before.js" file only containing the `require('react-native').unstable_enableLogBox();` line.
And then `import './before';` at the very top of index.js

**Comment by Daniel0liver** [comment]
> Não tenho certeza se isso é um conflito com a biblioteca do brinde.
> Mas suponho que você colocou o `require('react-native').unstable_enableLogBox();`arquivo no topo do arquivo index.js. Também não funcionou para mim.
> 
> O que funcionou para mim foi criar um arquivo "before.js" contendo apenas a `require('react-native').unstable_enableLogBox();`linha.
> E então, `import './before';`no topo do index.js

Yes, this is working for me. I have the same problem after adding lib react-native-tiny-toast to my project
